tests/vite-plugin-react-pages.ts

```diff
@@ -0,0 +1,16 @@
+import { runInRepo } from '../utils'
+import { RunOptions } from '../types'
+
+export async function test(options: RunOptions) {
+	await runInRepo({
+		...options,
+		repo: 'vitejs/vite-plugin-react-pages',
+		overrides: {
+			'@vitejs/plugin-react': true,
+		},
+		branch: options.viteMajor === 4 ? 'dev/v4' : 'main',
+		build: 'build:docs',
```

Comment:
is build:docs the right build  command? for vite-ecosystem-ci to be effective it would need to use the locally built vite-plugin-react-pages to serve the docs and test for actual plugin behavior / vite api usage.